src/attributes.md

```diff
@@ -0,0 +1,49 @@
+# Attributes
+
+Attributes come in two types - 'builtin'/'inert' attributes, and 'non-builtin'/'active' attributes:
```

Comment:
```suggestion
Attributes come in two types: *inert* (or *built-in*) and *active* (*non-builtin*).
```

Reply:
Could switch these around as well if we prefer. (`*built-in* (or *inert*)`)

I like italics for the first time a new term is introduced.